spec/models/group_spec.rb

```diff
@@ -125,6 +125,22 @@
     context 'with bob in the team' do
       before { team.people << bob }
 
+
+      context 'and bob is a team leader' do
+
+        before do
+          bob.memberships.first.update(leader: true)
+        end
+
+        it 'has .leaderships return array containing bob\'s membership' do
+          expect(team.leaderships.to_a).to eq [bob.memberships.first]
+        end
```

Comment:
The expectation now explictly refers to `bob.memberships.first` - hope this makes intent of spec clearer.